conf/application.conf

```diff
@@ -122,7 +88,7 @@ metrics {
 # Microservice specific config
 
 auditing {
-  enabled = true
+  enabled = false
```

Comment:
Why is auditing set to false?

Reply:
This is just to disable lots of log-spam when running tests.  It's overridden back to true in config for all the environments